twilight/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["-C", "target-cpu=native"]
```

Comment:
I am not sure if we should have this here, it could give issues with people compiling it then moving their bot to a vps where the same features are not available. But maybe we should have it written in the documentation for the feature.

Reply:
They also say this in the `simd-json` README because by definition, the crate needs modern CPU futures. Does `twilight` need to repeat this?

Reply:
oh i completely forgot that that config existed yeah we should remove it

Reply:
fixed it